### PR TITLE
DEV-3904-find-files-extension

### PR DESCRIPTION
**Pull request by zamazzal**
…exercise

### CODE REVIEW

**Review by nprimo — CHANGES_REQUESTED** [review]

File: sh/tests/find-files-extension_test.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
```
[the remark this answers is not in the document]
Comment (zamazzal) [reply sh/tests/find-files-extension_test.sh:7]: fixed

File: subjects/devops/find-files-extension/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+- You can use this for testing: https://assets.01-edu.org/devops-branch/find-files-extension-example.zip
+
+- What to use : `find`
+
+- The output should be exactly like the example bellow but with the expected name
+
+```console
+$pwd
+<..>/find-files-extension-example
```
[the remark this answers is not in the document]
Comment (zamazzal) [reply subjects/devops/find-files-extension/README.md:19]: I used pwd to indicate that the user should be inside the folder find-files-example

File: subjects/devops/find-files-extension/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+## find-files-extension
+
+### Instructions
+
+Create a file `find-files-extension.sh`, which will look, from the current directory and its sub-folders for:
```
[the remark this answers is not in the document]
Comment (zamazzal) [reply subjects/devops/find-files-extension/README.md:5]: I prefer this way cause it's serves the context better

**Review by nprimo — COMMENTED** [review]
Please, let me know if the `README.md` is supposed to be in this specific sub-directory instead of `public/subjects/`

**Review by nprimo — APPROVED** [review]
LGTM!

**Pull request merged**